tools/btmidi-server.c

```diff
@@ -0,0 +1,887 @@
+/*
+ *  BlueZ - Bluetooth protocol stack for Linux
+ *
+ *  Copyright (C) 2014  Google Inc.
```

Comment:
I'm sure you forgot to edit this line, ;)
It's ok, don't worry with this at this moment.